supernode/server/task_bridge.go

```diff
@@ -0,0 +1,82 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+	"github.com/dragonflyoss/Dragonfly/pkg/errortypes"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+)
+
+func (s *Server) registerTask(ctx context.Context, rw http.ResponseWriter, req *http.Request) (err error) {
+	reader := req.Body
+	request := &types.TaskCreateRequest{}
+	if err := json.NewDecoder(reader).Decode(request); err != nil {
+		return errors.Wrap(errortypes.ErrInvalidValue, err.Error())
+	}
+
+	if err := request.Validate(strfmt.NewFormats()); err != nil {
+		return errors.Wrap(errortypes.ErrInvalidValue, err.Error())
+	}
+
+	resp, err := s.TaskMgr.Register(ctx, request)
+	if err != nil {
+		return err
+	}
+	return EncodeResponse(rw, http.StatusCreated, resp)
```

Comment:
the current `dfget` cannot handle the response code `http.StatusCreated`

Reply:
I know that. And we should use the new API for dfget later. 😄